src/main/java/org/folio/service/transformationfields/TransformationFieldsServiceImpl.java

```diff
@@ -46,6 +42,7 @@
     transformationFields.addAll(buildTransformationFields(INSTANCE, TransformationConfigConstants.INSTANCE_FIELDS_CONFIGS, referenceData));
     transformationFields.addAll(buildTransformationFields(HOLDINGS, TransformationConfigConstants.HOLDINGS_FIELDS_CONFIGS, referenceData));
     transformationFields.addAll(buildTransformationFields(ITEM, TransformationConfigConstants.ITEM_FIELDS_CONFIGS, referenceData));
+    Collections.sort(transformationFields, (field1,field2) -> field1.getFieldId().compareTo(field2.getFieldId()));
```

Comment:
Not sure if it works the way we need it. I think the instance fields should be first, please ask Magda about that, because as I understand with this comparator, the holding fields are first

Reply:
Correct, I already confirmed with Magda, she agreed to leave the order this way for now
[3:14 PM] Magda Zacharska
    Let's leave this order for now. Once we hear from them that they want instances first followed by holdings and items  then we will adjust.

Reply:
ok, then I will remove those tests for equals responses from my PR.
Also, I think it is better to use List.sort instead of Collection.sort, we can write it as: transformationFields.sort(Comparator.comparing(TransformationField::getFieldId));

Reply:
Probably there is not much difference. Given that we will revisit this logic anyway once we get proper requirements, this will probably go away